docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -206,6 +211,39 @@ function processMessage(arg) {
 > [!NOTE]
 > The `showNotification` implementation is not shown in the sample code provided by this article. For an example of how you might implement this function within your add-in, see [Office Add-in Dialog API Example](https://github.com/OfficeDev/Office-Add-in-Dialog-API-Simple-Example).
 
+### Cross-domain messaging to the host runtime
+
+The dialog may be navigated away from the add-in's domain or the parent JavaScript runtime (either in a task pane or a UI-less runtime that hosts a function file) may be navigated away from the add-in's domain after the dialog is opened. If either of these things has happened, then a call of `messageParent` will fail unless your code specifies the domain of the parent runtime. You do this by adding a [DialogMessageOptions](/javascript/api/office/office.dialogmessageoptions) parameter to the call of `messageParent`. This object has a `targetOrigin` property that specifies the domain to which the message should be sent. If the parameter is not used, Office assumes that the target is the same domain that the dialog is currently hosting.
+
+> [!NOTE]
+> Using `messageParent` to send a cross-domain message requires the [Dialog Origin 1.1 requirement set](../reference/requirement-sets/dialog-origin-requirement-sets.md).
+
+The following is an example of using `messageParent` to send a cross-domain message.
+
+```js
+Office.context.ui.messageParent("Some message", { targetOrigin: "https://target.domain.com" });
+```
+
+> [!NOTE]
+> The `DialogMessageOptions` parameter was released approximately July 19th, 2021. For about 30 days after that date, in Office on the web, the first time that `messageParent` is called without the `DialogMessageOptions` parameter and the parent is a different domain from the dialog, the user will be prompted to approve sending data to the target domain. If the user approves, the user's answer is cached for 24 hours. The user is not prompted again during this period when `messageParent` is called with the same target domain.
```

Comment:
What about something like: 
In Office on the web, the user will be prompted to approve sending data to the target domain 30 days after `messageParent` is called without the `DialogMessageOptions` parameter and the parent is a different domain from the dialog?
I'm trying to figure out why the release date is necessary to publish (like, I get that it influences people who already have this code and fit the two conditions) but trying to avoid having to state it directly. 

"The user is not" -> isn't

Reply:
The prompt is only going to happen for an approximately 30 day period between July 19th and Aug 19th or thereabouts. We need a way to convey this approximate date range. So, we have to mention the start date.